harness/determined/deploy/gcp/terraform/modules/compute/main.tf

```diff
@@ -201,14 +131,23 @@ resource "google_compute_instance" "master_instance" {
 
     docker network create ${var.master_docker_network}
 
+    touch /usr/local/determined/etc/master.yaml
+    docker run \
```

Comment:
since this overhead is on every new install, we should make sure the image is very small even though it's short in comparison to the rest

Reply:
it's another binary in the same image, so it's slightly larger but we don't pull 2 images